src/Symfony/Component/Uid/Ulid.php

```diff
@@ -20,7 +20,7 @@
  */
 class Ulid extends AbstractUid
 {
-    private const NIL = '00000000000000000000000000';
+    protected const NIL = '00000000000000000000000000';
```

Comment:
What about moving it to `NilUlid` as public const instead of making it protected here?

Reply:
Well, consistency with https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Uid/NilUuid.php I guess